src/_includes/docs/add-to-app/ios-project/embed-split.md

```diff
@@ -0,0 +1,71 @@
+### Use frameworks in Xcode and Flutter framework as podspec {:#method-c .no_toc}
+
+#### Approach {:#method-c-approach}
+
+This method generates Flutter as a CocoaPods podspec instead of
+distributing the large `Flutter.xcframework` to other developers,
+machines, or continuous integration systems.
+Flutter still generates the `App.xcframework` and plugin frameworks
+as described in Option B.
```

Comment:
It might not be clear what option B is, consider something like:

```
as described in the second option, [Use frameworks](#link-to-second-tab)
```